### PR TITLE
removed documentation about responsive column resets and clearfix

### PR DESCRIPTION
**Pull request by rcarlis**

### CODE REVIEW

File: style-and-behavior/utility-helpers.html [hunk 1]
```diff
@@ -7,7 +7,6 @@
 {% include /style-and-behavior/utility-helpers/quick-floats.html %}
```
Comment (wjramos) [inline style-and-behavior/utility-helpers.html:7]: @mhewson -- is this something that should be removed?

File: _includes/style-and-behavior/utility-helpers/clearfix.html [hunk 2]
```diff
@@ -1,26 +0,0 @@
-<section class="vertical-push-3">
```
Comment (mhewson) [inline _includes/style-and-behavior/utility-helpers/clearfix.html:1]: at this point this is still needed for other patterns, we just need to remove this from grid docs but not from the docs all together

File: _data/navigation.yml [hunk 3]
```diff
@@ -471,9 +468,6 @@ items:
                 - label: Center Content Blocks
                   href: helper-classes-center
 
-                - label: Clearfix
```
Comment (mhewson) [inline _data/navigation.yml:474]: at this point this is still needed for other patterns, we just need to remove this from grid docs but not from the docs all together

File: _includes/layouts/grid/grid.html [hunk 4]
```diff
@@ -261,50 +261,6 @@ <h3 id="grid-example-wrapping">Example: Column wrapping</h3>
 </div>
 {% endhighlight %}
 
-  <h3 id="grid-responsive-resets">Responsive column resets</h3>
```
Comment (mhewson) [inline _includes/layouts/grid/grid.html:264]: nice

File: _includes/style-and-behavior/utility-helpers/clearfix.html (hunk 2 above)
Comment (rcarlis) [inline _includes/style-and-behavior/utility-helpers/clearfix.html:1]: gotcha, I'll add the clearfix doc back in

**Comment by wjramos** [comment]
Looks good to me -- I approve

[Approved with PullApprove](https://pullapprove.com/rei/rei-cedar/pull-request/66/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=rei/rei-cedar)

**Pull request merged**